lib/rack/cache/context.rb

```diff
@@ -38,7 +38,11 @@ def metastore
     # value effects the result of this method immediately.
     def entitystore
       uri = options['rack-cache.entitystore']
-      storage.resolve_entitystore_uri(uri)
+      if uri.nil?
```

Comment:
It's more concise but the intent is less obvious, in my opinion.

However if you prefer it, I'll change it.